stdlib/json/0/json.rbs

```diff
@@ -403,3 +403,227 @@ class Array[unchecked out Elem]
   #
   def to_json: (?JSON::State state) -> String
 end
+
+class BigDecimal
+  # Import a JSON Marshalled object.
+  #
+  # method used for JSON marshalling support.
+  #
+  def self.json_create: (Hash[String, String] object) -> instance
+
+  # Marshal the object to JSON.
+  #
+  # method used for JSON marshalling support.
+  #
+  def as_json: (*untyped) -> Hash[String, String]
```

Comment:
[question] Is there a better way than `*untyped`?

Reply:
It's not very clear... `*untyped` would be a good solution. 🙄